### PR TITLE
Added acceptable character check between username and karma action

### PR DESCRIPTION
**Pull request by rosslebeau**

### CODE REVIEW

File: Sources/Camille/KarmaBot.swift [hunk 1]
```diff
@@ -33,21 +33,24 @@ struct KarmaBotOptions {
...
     
     init(
         targets: [String]? = nil,
         addText: String? = nil,
         addReaction: String? = nil,
         removeText: String? = nil,
         removeReaction: String? = nil,
-        textDistanceThreshold: Int = 4
+        textDistanceThreshold: Int = 4,
+        allowedBufferCharacters: [Character] = [Character(" "), Character(":")]
```
Comment (IanKeen) [inline Sources/Camille/KarmaBot.swift]: Take advantage of inference here: `[" ", ":"]`

Comment (rosslebeau) [reply Sources/Camille/KarmaBot.swift]: Done. GH used to show that comments were "outdated", what happened to this?

**Comment by IanKeen** [comment]
I like it!, I _was_ pondering if we should use `CharacterSet` instead but I've had some earlier problems with it on linux so I think its not super important right now 😁

**Comment by aranasaurus** [comment]
Looks good to me!

File: Sources/Camille/KarmaBot.swift (hunk 1 above)
Comment (IanKeen) [reply Sources/Camille/KarmaBot.swift]: new features! it keeps threads alive and you can do smaller code reviews within a PR now too :D

**Comment by bugKrusha** [comment]
Cool! I likes.

**Comment by jamescmartinez** [comment]
:shipit:

**Pull request merged**